assets/js/base/components/price-slider/index.js

```diff
@@ -257,18 +265,16 @@ const PriceSlider = ( {
 								'Filter products by minimum price',
 								'woo-gutenberg-products-block'
 							) }
-							value={
```

Comment:
Removing value and change values dynamically allows keyboard input to function.

Reply:
This is a bit of an antipattern in React because you are changing things imperatively instead of declaratively (see ["When to Use Refs"](https://reactjs.org/docs/refs-and-the-dom.html#when-to-use-refs)). In the case of inputs it's usually okay to use `refs` to get various attributes of the input when it changes but not to _set values_ via the ref.

I understand that this might have appeared as a way to fix allowing the keyboard input to function but are we sure this is the only way to fix that?

I notice through this review that we're mutating other ref properties here for the input as well (zIndex on style) that should be done through react props instead of through the ref.